cmsplugin_filer_image/templates/cmsplugin_filer_image/image.html

```diff
@@ -1,46 +1,50 @@
-  {% with rnd='imagePlugin_'|add:d1|add:d2|add:d3|add:d4 %}
+{% addtoblock "js" %}
+{% if event_tracking %}
+<script type="text/javascript">
+ (function($) {
+   $(document).ready(function(){
+     var event = ['_trackEvent', '{{ event_tracking.category }}', '{{ event_tracking.action}}', '{{ event_tracking.label }}'];
+     $("#{{ rnd }} .image-link").off('click').click(function() {
+       _gaq.push(event);
+     });
+   });
+ })(jQuery)
+</script>
+{% endif %}
 
-    {% addtoblock "js" %}
-      {% if link and instance.enable_event_tracking and instance.event_category and instance.event_action %}
-        <script type="text/javascript">
-            (function($){
-              $(document).ready(function(){
-                $("#{{rnd}} .image-link").off('click').click(function(){_gaq.push(['_trackEvent', '{{instance.event_category}}', '{{instance.event_action}}', '{% if instance.event_label %}{{instance.event_label}}{% endif %}'])})
-              });
-            })(jQuery)
-        </script>
-      {% endif %}
+{% if overlay_link %}
+<script type="text/javascript">
+ (function($){
+   $('#{{ rnd }}').click(function(){
+     $('body').append(
+       '<div class="preview-image-overlay"><span> </span>' +
+       '<img src="{{ overlay_link }}" /></div>');
+     $('.preview-image-overlay').click(function() {$('.preview-image-overlay').remove();});
+   });
+ })(jQuery);
+</script>
+{% endif %}
+{% endaddtoblock %}
 
-      {% if overlay_link %}
-        <script type="text/javascript">
-            (function($){
-                $('#{{rnd}}').click(function(){
-                    var imgURL = $(this).data('rel');
-                    $('body').append(
-                        '<div class="preview-image-overlay">'+
-                            '<span> </span>'+
-                            '<img src="' + imgURL + '" />'+
-                        '</div>');
-                    $('.preview-image-overlay').click(function() {$('.preview-image-overlay').remove();} );
-                });
-            })(jQuery)
-        </script>
-      {% endif %}
-    {% endaddtoblock %}
-
-    {% if instance.image %}
-        {% if instance.image.width == opts.size.0 and instance.image.height == opts.size.1 %}
-            <!-- the width/height chosen as options are same with the original image width/height -->
-            {% include "cmsplugin_filer_image/img_element.html" with sized_image=instance.image %}
-        {% else  %}
-            {% thumbnail instance.image size crop=opts.crop upscale=opts.upscale subject_location=opts.subject_location as thumbnail %}
-            {% include "cmsplugin_filer_image/img_element.html" with sized_image=thumbnail %}
-        {% endif %}
-    {% endif %}
-  {% endwith%}
-{% endwith %}
+{% if image_url %}
+<span id="{{ rnd }}" class="filer_image_container {{ container_classes }}" {{ container_attributes }}
```

Comment:
minor: 
classes should use dashed names: filer_image_container -> filer-image-container